docs/user/api/integrity.md

```diff
@@ -0,0 +1,129 @@
+# Integrity API
+
+<!--[[ preview('user-api-docs') ]]-->
+
+The Integrity API provides access to PyPI's implementation of [PEP 740].
+
+## Concepts
+
+The concepts and objects in the Integrity API closely mirror [PEP 740]:
+
+* **Attestation objects** encapsulate a single "attestation" for a single file,
+  such as a [publish attestation] or [SLSA Provenance].
+
+* **Provenance objects** encapsulate *one or more* attestations for a given
+  file, bundling them with the *identity* that produced them.
+
+The Integrity API deals in provenance objects; users should extract and verify
+individual attestations from a file's provenance, as appropriate.
+
+## Routes
+
+### `GET /integrity/<project>/<version>/<filename>/provenance`
+
+Get the provenance object for the given `<filename>`.
+
+!!! note
+
+    The `<project>` and `<version>` components must be consistent with the
+    `<filename>`,  meaning that
+    `GET /integrity/sampleproject/1.0.0/unrelated-1.0.0.tar.gz/provenance`
+    will **not** work.
+
+This endpoint is currently only available as JSON.
+
+Example JSON request (default if no `Accept` header is passed):
+
+```http
+GET /integrity/sampleproject/1.0.0/sampleproject-1.0.0.tar.gz/provenance HTTP/1.1
+Host: pypi.org
+Accept: application/vnd.pypi.integrity.v1+json
+```
+
+??? note "Example JSON response"
+
+    This is an example response, demonstrating a provenance object containing
+    one attestation and its Trusted Publishing identity.
```

Comment:
Again, would. nice to be able to just use a real response from `sampleproject` here.

Reply:
True! I can send a PR to update `sampeproject` 🙂